src/faultlog/guard_with_eid_records.cpp

```diff
@@ -231,30 +329,85 @@ void GuardWithEidRecords::populate(sdbusplus::bus::bus& bus,
 
                 sectionJson["Callout Count"] = 1;
                 sectionJson["Callouts"] = jsonCallout;
-                jsonErrorLog["PLID"] =
-                    std::to_string(hwasState.deconfiguredByEid);
+                std::stringstream ss;
+                ss << std::hex << "0x" << hwasState.deconfiguredByEid;
+                jsonErrorLog["PLID"] = ss.str();
+                plid = hwasState.deconfiguredByEid;
                 jsonErrorLog["Callout Section"] = sectionJson;
                 jsonErrorLog["SRC"] = 0;
                 jsonErrorLog["DATE_TIME"] = "00/00/0000 00:00:00";
             }
 
-            jsonErrorLogSection.push_back(std::move(jsonErrorLog));
-            json jsonEventData = json::object();
-            jsonEventData["RESOURCE_ACTIONS"] = std::move(jsonResource);
-            jsonErrorLogSection.push_back(jsonEventData);
+            // populate resource actions section
+            json jsonResource = json::object();
+            jsonResource["TYPE"] = pdbgTargetName(guardedTarget.target);
+            std::string state = stateDeconfigured;
+            if (hwasState.functional)
+            {
+                state = stateConfigured;
+            }
+            jsonResource["CURRENT_STATE"] = std::move(state);
 
-            json jsonErrlogObj = json::object();
-            jsonErrlogObj["CEC_ERROR_LOG"] = std::move(jsonErrorLogSection);
+            jsonResource["REASON_DESCRIPTION"] =
+                getGuardReason(guardRecords, *physicalPath);
+
+            jsonResource["GARD_RECORD"] = true;
```

Comment:
Is it supposed to GUARD_RECORD? recently remember fixing the spelling for something similar.  Please check